modules/monitoring/application/controllers/TacticalController.php

```diff
@@ -92,9 +92,11 @@ public function indexAction()
             ->addSlice($summary->services_unknown_unhandled, array('class' => 'slice-state-unknown'))
             ->addSlice($summary->services_pending, array('class' => 'slice-state-pending'))
             ->addSlice($summary->services_pending_not_checked, array('class' => 'slice-state-not-checked'))
-            ->setLabelBig($summary->services_critical_unhandled)
-            ->setLabelBigEyeCatching($summary->services_critical_unhandled > 0)
-            ->setLabelSmall($this->translate('Services Critical'));
+            ->setLabelBig($summary->services_critical_unhandled ?: $summary->services_unknown_unhandled)
+            ->setLabelBigEyeCatching($summary->services_critical_unhandled + $summary->services_unknown_unhandled > 0)
+            ->setLabelSmall($summary->services_critical_unhandled > 0 || $summary->services_unknown_unhandled < 1
```

Comment:
Can't you just use `$summary->services_unknown_unhandled > 0 ? $this->translate('Services Unknown') : $this->translate('Services Critical')`?

Reply:
Imagine there are some critical and some unknown. And now have a look at what I pass to setLabelBig()... TL;DR: no, not really.

Reply:
I see.